openid-4-verifiable-presentations-1_0.md

```diff
@@ -620,6 +626,7 @@ The following is a non-normative example of a request object:
   "presentation_definition": {...},
   "nonce": "n-0S6_WzA2Mj",
   "wallet_nonce": "qPmxiNFCR3QTm19POc8u",
+  "wallet_attestation": true,
```

Comment:
ok, I want to find a solution but, sincerely, this appears very naif.

different trust frameworks may defines many different wallet attestations. By only requesting a "wallet attestation" (any of them, at least one) would end in a useless/weak parameter.

The RP needs to ask one or more (considering at least one of a set of supported one) wallet attestation.

For this reason, using presentation_definition or duckle appeared to me the best way to fully specify the wallet attestation, by type and properties.